src/components/SecondaryPanes/index.js

```diff
@@ -461,6 +479,18 @@ class SecondaryPanes extends Component<Props, State> {
   }
 }
 
+// Checks if user is in debugging mode and adds a delay preventing
+// excessive vertical 'jumpiness'
+function getRenderWhyPauseDelay(state, thread) {
+  const inPauseCommand = !!getPauseCommand(state, thread);
+
+  if (!inPauseCommand) {
+    return 100;
+  }
+
+  return 0;
```

Comment:
is this delay new?

Reply:
No, I haven't changed it since creating the initial PR.